earthly/docs/Earthfile

```diff
@@ -0,0 +1,92 @@
+VERSION 0.7
+FROM python:3.11-bullseye
+
+mkdocs-material:
+    # Derived from official mkdocs-material docker container.
+    # https://github.com/squidfunk/mkdocs-material/blob/master/Dockerfile
+
+    # Install poetry and our python dependencies.
+    DO ../python+POETRY_SETUP --extra_file=start.sh
```

Comment:
Passing `start.sh` as an extra file is a bit misleading here as it doesn't actually do anything to change the `poetry install` invocation. I think it's a bit more clear to just have it be its own `COPY` line.